Essentials/src/com/earth2me/essentials/commands/EssentialsCommand.java

```diff
@@ -324,6 +343,13 @@ private static boolean canInteractWith(User interactor, User interactee) {
         return commands;
     }
 
+    /**
+     * Formats the given commandLabel to be friendly readable.
+     */
+    public String formatCommand(String commandLabel, String[] args) {
```

Comment:
```suggestion
    /**
     * Formats the given commandLabel to be friendly readable.
     */
    protected final String formatCommand(String commandLabel, String[] args) {
```

This shouldn't be exposed outside this class and command implementation classes.